plugins/auth-backend/src/identity/router.ts

```diff
@@ -68,7 +72,40 @@ export function bindOidcRouter(
     res.status(501).send('Not Implemented');
   });
 
-  router.get('/v1/userinfo', (_req, res) => {
-    res.status(501).send('Not Implemented');
+  // This endpoint doesn't use the regular HttpAuthService, since the contract
+  // is specifically for the header to be communicated in the Authorization
+  // header, regardless of token type
+  router.get('/v1/userinfo', async (req, res) => {
+    const token = req.headers.authorization?.replace(/^Bearer\s+/i, '');
```

Comment:
Shouldn't we be a bit more strict? :P I'd rather match on `/^Bearer ([\w.]+)$/i` or something and use the captured group

Reply:
Yep I can switch it over to what we have in other places

Reply:
Copied over the implementation from the http auth service